lib/node_modules/@stdlib/string/remove-first/lib/remove_first.js

```diff
@@ -55,7 +56,7 @@ function removeFirst( str ) {
 	if ( str === '' ) {
 		return '';
 	}
-	return str.substring( 1 );
+	return str.substring( nextGraphemeClusterBreak( str, 0 ) );
```

Comment:
Quick sanity check: what happens if `nextGraphemeClusterBreak` returns `-1` here?

Reply:
It won't ever return `-1`

Reply:
The possibility of `-1` is predicated on the resolution of the discussion above.

Reply:
Handled the cases when `nextGraphemeClusterBreak` will return `-1`. It will either be when the input string is empty or has only 1 extended grapheme cluster, and for both of these cases, `removeFirst` should return an empty string.